alts/worker/runners/base.py

```diff
@@ -463,7 +477,9 @@ def write_to_file(file_base_name: str, artifacts_section: dict):
                 for inner_artifact_key, inner_content in content.items():
                     log_base_name = f'{TESTS_SECTION_NAME}_{inner_artifact_key}'
                     write_to_file(log_base_name, inner_content)
-
+            elif artifact_key == 'initialize_terraform':
+                stdout = content['stdout']
+                content['stdout'] = f'Task ID: {self._task_id}\n\n{stdout}'
```

Comment:
it seems like we will skip `initialize_terraform.log` from compressing and uploading

Reply:
Fixed